### DISCUSSION TITLE
problem with v8.0 and v8.3

### DISCUSSION THREAD

**Discussion by Chomemel** [sanni/cartreader#377] [Need help]
Hi,

I tried to flash my arduino, with v7.3 ; 8.0 and 8.3 in that order:
- 8.3 (failed)
- 7.3 (succeeded)
- 8.3 again (failed)
- 8.0 (failed)

Every time it failed, the error occured during the compliation, with always he same error:
**avr-g++: error: device-specs/specs-atmega2560: No such file or directory
exit status 1
Error compiling for board Arduino Mega or Mega 2560**

Any idea?

**Comment by sanni** [comment]
Are you on a Windows PC and are you using the portable Arduino IDE from the releases tab?
If yes, have you tried running the arduino.exe as Admin? Right-click on it and then choose Run as administrator.

The file it's not finding is located here: V8.3_Portable\Arduino IDE Portable\hardware\tools\avr\lib\gcc\avr\7.3.0\device-specs\specs-atmega2560
Not sure why it would not find it. Maybe try another PC.

**Comment by Chomemel** [comment]
Hi Sanni,
Fist of all, thanks for all the good work and the time you take to answer us.
Thanks to your precious help I found out the file **_device-specs_** was missing. Turns out my archiver would not extract this file lately for some obscure reason. Anyway, I tried with another archiver, and it worked perfectly.
Case solved.
Can't wait to buid the cart reader.
Thanks again for the quick answer.

**Comment by dirn** [comment]
I'm having this same problem with V9.1.

```
avr-g++: error: device-specs/specs-atmega2560: No such file or directory
exit status 1
Error compiling for board Arduino Mega or Mega 2560.
```

I, however, have the file where you say it should be. I've tried running the included arduino.exe both regularly and as Administrator.

<img width="693" alt="image" src="https://user-images.githubusercontent.com/168109/179639823-4e31f621-af01-4525-941a-34487b7b907e.png">

Is there anything else I can try?

**Comment by sanni** [comment]
You can use the arduino-1.8.19-windows.exe from here https://www.arduino.cc/en/software to install Arduino IDE.
And then copy the "libraries" and "Cart_Reader" folder from "V9.1_Portable\Arduino IDE Portable\portable\sketchbook" to "C:\Users\[your user name]\Documents\Arduino" 
Finally launch "Arduino" from the start menu.

**Comment by dirn** [comment]
I tried that and ran into problems with the compiler being able to find (at least) one of the header files. I decided to start over. I had unzipped the release in my Downloads folder. This time I copied the Arduino IDE Portable folder into C:\Users\<me> and it worked!
